deploy/cephfs/kubernetes/csi-nodeplugin-rbac.yaml

```diff
@@ -3,51 +3,3 @@ apiVersion: v1
 kind: ServiceAccount
```

Comment:
Should we rename this file from `csi-nodeplugin-rbac.yaml` to ` csi-nodeplugin-sa.yaml` or even better yet just merge this content into `csi-cephfsplugin.yaml`? (comment/thought applies to the rbd counterpart files as well)

Reply:
I thought of doing it but the reason for not doing it is if we change the filename same changes need to be reflected in E2E and install script in example folder also. in feature, if we add new roles to node plugin we need to revert back the same changes. to avoid this keeping the filename as it it for now.

Reply:
I understand that more files need to be touched, but IMHO as the deployment manifests are one of the first user facing files, it is better to be clear in this regard.

Should we track it for renaming before the next release, such that if there are no RBAC changes till then, we will rename (or remove) this file and make required e2e, script and doc changes?

Also, does this removal require upgrade related documentation?

Reply:
Yes, I will let the issue be open, and also upgrade documentation will be updated during release time.